filebeat/templates/daemonset.yaml

```diff
@@ -82,7 +94,7 @@ spec:
       {{- end }}
       - name: data
         hostPath:
-          path: {{ .Values.hostPathRoot }}/{{ template "filebeat.fullname" . }}-{{ .Release.Namespace }}-data
+          path: {{ .Values.hostPathRoot }}/{{ template "filebeat.fullname" . }}-data
```

Comment:
Sure.
Should have been an missconception of me that the unit tests where failing because of this.
I also remember modifying the value to the test file.

Yet, looks like it should be reverted

Reply:
Inverted, pytest'd
https://github.com/elastic/helm-charts/pull/964/commits/c548a20f1e6bcdf40a2986bc0a4fcf63f90d674e